packages/server/lib/browsers/firefox.ts

```diff
@@ -443,16 +451,30 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
   const [
     foxdriverPort,
     marionettePort,
-    geckoDriverPort,
     webDriverBiDiPort,
-  ] = await Promise.all([getPort(), getPort(), getPort(), getPort()])
+  ] = await Promise.all([getPort(), getPort(), getPort()])
 
   defaultLaunchOptions.preferences['devtools.debugger.remote-port'] = foxdriverPort
   defaultLaunchOptions.preferences['marionette.port'] = marionettePort
 
   // NOTE: we get the BiDi port and set it inside of geckodriver, but BiDi is not currently enabled (see remote.active-protocols above).
   // this is so the BiDi websocket port does not get set to 0, which is the default for the geckodriver package.
-  debug('available ports: %o', { foxdriverPort, marionettePort, geckoDriverPort, webDriverBiDiPort })
+  debug('available ports: %o', { foxdriverPort, marionettePort, webDriverBiDiPort })
+
+  const profileDir = utils.getProfileDir(browser, options.isTextTerminal)
+
+  // Delete the profile directory if in open mode.
```

Comment:
```suggestion
  // Delete the legacy profile directory if in open mode.
```

Are there any concerns with this where people might have one project on one version of Cypress and one on another?

Reply:
addressed in e8ab563b78